components/ruby/facter/Makefile

```diff
@@ -39,25 +40,20 @@ COMPONENT_CLASSIFICATION=	Development/Ruby
 COMPONENT_LICENSE=	Apache v2.0
 COMPONENT_LICENSE_FILE=	facter.license
 
+include $(WS_TOP)/make-rules/common.mk
+include $(WS_TOP)/make-rules/gem.mk
```

Comment:
With `BUILD_STYLE= gem` you should not need the `gem.mk` include.  Or, is it really needed?

Reply:
done

Reply:
If I remove common.mk gmake clobber fails and if I remove ruby.mk gmake REQUIRED_PACKAGES fails